CjCollins-Plano.md

```diff
@@ -0,0 +1,6 @@
+# Plano Tx
+
+## Places to Eat
+- Cozy Thai
```

Comment:
If you like it really hot, ask for Thai spicy :) Medium spicy is almost too much for most ;)